example_makefiles/makefile.inc.Mac.brew

```diff
@@ -77,17 +77,11 @@ BLASLDFLAGS=-framework Accelerate
 
 SWIGEXEC=/usr/local/bin/swig
 
-# The Python include directories for the current python executable can
-# typically be found with
-#
-# python -c "import distutils.sysconfig; print distutils.sysconfig.get_python_inc()"
-# python -c "import numpy ; print numpy.get_include()"
-#
-# the paths below are for the system python (not the macports one)
-
-PYTHONCFLAGS=-I/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 \
--I/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/numpy/core/include
+# The Python include directories for the current python executable
 
+PYTHON_INC = $(shell python -c "from __future__ import print_function; import distutils.sysconfig; print(distutils.sysconfig.get_python_inc())")
+NUMPY_INC = $(shell python -c "from __future__ import print_function; import numpy ; print(numpy.get_include())")
```

Comment:
you don't need the from __future__ because the print has just a single argument, ie. `print (x)` woks fine in py2 as well.